pkgs/applications/editors/vscode/extensions/natqe.reload/default.nix

```diff
@@ -0,0 +1,19 @@
+{
+  lib,
+  vscode-utils,
+}:
+vscode-utils.buildVscodeMarketplaceExtension {
+  mktplcRef = {
+    name = "reload";
+    publisher = "natqe";
+    version = "0.0.7";
+    hash = "sha256-j0Dj7YiawhPAMHe8wk8Ph4vo26IneidoGJ4C9O7Z/64=";
+  };
+  meta = {
+    description = "This extension will add reload button to status bar in the right-bottom of your VSCode editor.";
+    downloadPage = "https://marketplace.visualstudio.com/items?itemName=natqe.reload";
+    homepage = "https://github.com/natqe/reload";
+    license = lib.licenses.unfree;
```

Comment:
I don't think unfree is the right license here, have you opened an issue upstream to get some clarification from the extension's author?

Reply:
> i thought if no License was provided it automatically is unfree.

I'm not aware of that, but thanks for opening the issue. I'll wait a bit before merging this, if this is OK for you.